### PR TITLE
Add ability to use existing master/node service accounts

### PR DESCRIPTION
**Pull request by Sebor**
Fixes https://github.com/terraform-yc-modules/terraform-yc-kubernetes/issues/23

### CODE REVIEW

**Comment by Sebor** [comment]
@romati88 @pauljamm hi!
Could you review the PR?

File: kms.tf [hunk 1]
```diff
@@ -13,10 +13,19 @@ resource "yandex_kms_symmetric_key" "kms_key" {
 }
 
 resource "yandex_kms_symmetric_key_iam_binding" "encrypter_decrypter" {
-  count            = var.create_kms ? 1 : 0
+  count            = var.create_kms && !local.create_sa ? 1 : 0
   symmetric_key_id = yandex_kms_symmetric_key.kms_key[count.index].id
   role             = "kms.keys.encrypterDecrypter"
   members = [
-    "serviceAccount:${yandex_iam_service_account.master.id}",
+    try("serviceAccount:${yandex_iam_service_account.master[0].id}", null),
```
Comment (romati88) [inline kms.tf:20]: why you use try() here?

File: iam.tf [hunk 2]
```diff
@@ -3,62 +3,66 @@ locals {
     service_account_name = "k8s-service-account-${random_string.unique_id.result}"
     node_account_name    = "k8s-node-account-${random_string.unique_id.result}"
   }
+  create_sa = var.use_existing_sa && var.master_service_account_id != null && var.node_service_account_id != null ? true : false
```
Comment (romati88) [inline iam.tf:6]: may be try to use variable validation on var.use_existing_sa

File: kms.tf (hunk 1 above)
Comment (Sebor) [reply kms.tf]: It was typo. Fixed

File: iam.tf (hunk 2 above)
Comment (Sebor) [reply iam.tf:6]: Hi @romati88!
What do you mean? We cannot use another variable inside variable validate section. That's why I use locals

**Review by romati88 — APPROVED** [review]

**Pull request merged**